docs/src/main/sphinx/connector/memory.rst

```diff
@@ -78,7 +63,24 @@ DROP TABLE
 
 Upon execution of a ``DROP TABLE`` operation, memory is not released
 immediately. It is instead released after the next write operation to the
-connector.
+catalog.
+
+.. _memory_dynamic_filtering:
```

Comment:
This section just looks like it changed because I moved the SQL support section up above it..